.github/workflows/continuous-integration.yaml

```diff
@@ -10,6 +10,42 @@ on:
       - master
 
 jobs:
+  fix-formatting:
+    if: >-
+      github.actor != 'github-actions[bot]' &&
+      github.event_name == 'pull_request' &&
+      github.repository == 'nock/nock'
+    name: Fix formating
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: true
+          ref: ${{ github.head_ref }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Cache npm dependencies
+        uses: actions/cache@v1
+        with:
+          key: npm-${{ hashFiles('package-lock.json') }}
+          path: ~/.npm
+          restore-keys: |
+            npm-
+      - name: Install dependencies
+        run: npm ci --ignore-scripts --no-audit --no-progress --prefer-offline
+      - name: Format
+        run: npm run format:fix
+      - name: Commit
+        run: |
+          git config user.email "actions@github.com"
+          git config user.name "GitHub Actions"
+          git add .
+          git commit -m "chore: format"
+          git push origin HEAD:${{ github.head_ref }}
```

Comment:
I'm not sure if this will work, it will likely fail with an unauthenticated error. Instead of origin do `https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.gi`

Reply:
Ohh okay TIL: `persist-credentials: true`